NiazMorshed2007/shadcn-tiptap
# Adds mobile support for image resize handles

## Description

The image resize handles before were relying on mouse events to work. This is fine on desktop, but breaks on mobile which meant image resizing was not working properly.

## Proposed Changes

- Added touch events to the `image` extensions that work in conjunction with the mouse events already present
- Updates the image dimensions based on these touch events


## Steps to Test

- Using your browser's developer tools, go to a simulated mobile breakpoint and use the resize handles on the image. The dimensions should properly adjust. Whereas before the drag handles did nothing.
